Content.Server/Wieldable/Components/WieldableComponent.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using Content.Shared.Sound;
+using Content.Shared.Verbs;
+using Robust.Shared.Analyzers;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+
+namespace Content.Server.Wieldable.Components
+{
+    /// <summary>
+    ///     Used for objects that can be wielded in two or more hands,
+    /// </summary>
+    [RegisterComponent, Friend(typeof(WieldableSystem))]
+    public class WieldableComponent : Component
+    {
+        public override string Name => "Wieldable";
+
+        [DataField("wieldSound")]
+        public SoundSpecifier? WieldSound = new SoundPathSpecifier("/Audio/Effects/thudswoosh.ogg");
+
+        [DataField("unwieldSound")]
+        public SoundSpecifier? UnwieldSound = default!;
+
+        /// <summary>
+        ///     Number of free hands required (excluding the item itself) required
+        ///     to wield it
+        /// </summary>
+        [DataField("freeHandsRequired")]
+        public int FreeHandsRequired = 1;
+
+        public bool Wielded = false;
+
+        public string WieldedInhandPrefix = "wielded";
+
+        public string? OldInhandPrefix = null;
+
+        [DataField("wieldTime")]
+        public float WieldTime = 0.5f;
+
+        [Verb]
+        public sealed class ToggleWieldVerb : Verb<WieldableComponent>
+        {
+            public override bool AlternativeInteraction => true;
+
+            protected override void GetData(IEntity user, WieldableComponent component, VerbData data)
+            {
+                data.Visibility = VerbVisibility.Visible;
```

Comment:
Maybe hide this verb unless the axe is actually in the users hand, e.g. `commponent.Wielded ||  CanWield()`?
Or maybe make the `Activate()` try to pick-up and wield the item if it's not in hand.

Reply:
i think its better for discoverability if it shows up regardless but just doesn't let you actually activate it, but i'm not sure

Reply:
Isn't Visible the default?